.github/workflows/ci.yaml

```diff
@@ -112,7 +112,7 @@ jobs:
         pytest --doctest-modules openmmforcefields --ignore=openmmforcefields/tests
 
     - name: Upload coverage report to CodeCov
-      uses: codecov/codecov-action@v3.1.1
+      uses: codecov/codecov-action@v3.1.2
```

Comment:
Recommend `uses: codecov/codecov-action@v3` which has the effect of these PRs without the noise. Still makes the bot open a PR for major version bumps, since in principal that's the only time things might break.